integration/resources/templates/combination/connector_appsync_to_eventbus.yaml

```diff
@@ -0,0 +1,173 @@
+    Properties:
+      Source:
+        Id: AppSyncEventBusDataSource
+      Destination:
+        Id: EventBus
+      Permissions:
+      - Write
+
+  ApiKey:
+    Type: AWS::AppSync::ApiKey
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+
+  TriggerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Role: !GetAtt EventBridgeRole.Arn
+      Environment:
+        Variables:
+          API_KEY: !GetAtt ApiKey.ApiKey
+          GRAPHQL_URL: !GetAtt AppSyncApi.GraphQLUrl
+          EventBusName: !Ref EventBus
+      Runtime: nodejs16.x
+      Handler: index.handler
+      InlineCode: |
+        const https = require("https");
+        const AWS = require('aws-sdk');
+
+        exports.handler = async () => {
+          const queries = {
+            sayHello: /* GraphQL */ `
+              query {
+                sayHello
+              }
+            `,
+          };
+
+          const fetch = async (url, options) =>
+            new Promise((resolve, reject) => {
+              const req = https.request(url, options, (res) => {
+                const body = [];
+                res.on("data", (chunk) => body.push(chunk));
+                res.on("end", () => {
+                  const resString = Buffer.concat(body).toString();
+                  resolve(resString);
+                });
+              });
+              req.on("error", (err) => {
+                reject(err);
+              });
+              req.on("timeout", () => {
+                req.destroy();
+                reject(new Error("Request time out"));
+              });
+              req.write(options.body);
+              req.end();
+            });
+
+          const makeRequest = async (queryName) => {
+            const options = {
+              method: "POST",
+              headers: {
+                "x-api-key": process.env.API_KEY,
+              },
+              body: JSON.stringify({ query: queries[queryName] }),
+              timeout: 10000, // ms
+            };
+
+            const response = await fetch(process.env.GRAPHQL_URL, options);
+            let body = JSON.parse(response);
+            const data = body.data?.[queryName];
+            
+            if (body.errors !== undefined) {
+              throw JSON.stringify(body.errors);
+            }
+            
+            return body.data;
+          };
+
+          const sayHelloResponse = await makeRequest("sayHello");
+
+          var put_params = {
+            Entries: [
+              {
+                EventBusName: process.env.EventBusName,
+                Source: "com.mycompany.myapp",
+                Detail:"{ \"key1\": \"value1\", \"key2\": \"value2\" }",
+                Resources:[
+                  sayHelloResponse['sayHello']
+                ],
+                DetailType:"myDetailType"
+              }
+            ]
+          }
+
+          var eventbridge = new AWS.EventBridge();
+          await eventbridge.putEvents(put_params).promise();
```

Comment:
The test expects asn explicit [response](https://github.com/ssenchenko/serverless-application-model/blob/00245b65ca324a418ebe713359350fab829b964a/integration/combination/test_connectors.py#L73)

We need to process the result, check that there are no errors and return 200. Or return errors too (see the test)

Reply:
I'll take a look at ways we can verify that the event is actually in the eventBus after calling `putEvents()` and add that to this test case.

Reply:
Updated the test case to verify that events are succesfully sent to the eventBus via the putEvents call